t/lib/OpenQA/Test/FullstackUtils.pm

```diff
@@ -198,11 +205,7 @@ sub wait_for_developer_console_available {
     pass("os-autoinst cmd srv available after $seconds seconds");
 
     # check initial connection
-    OpenQA::Test::FullstackUtils::wait_for_developer_console_contains_log_message(
-        $driver,
-        qr/Connection opened/,
-        'connection opened'
-    );
+    OpenQA::Test::FullstackUtils::wait_for_developer_console_like($driver, qr/Connection opened/, 'connection opened');
```

Comment:
You can get rid of `OpenQA::Test::FullstackUtils::` here as well.

Reply:
done